src/main/resources/application-dev-gcp.yml

```diff
@@ -19,3 +19,11 @@ KAFKA_SCHEMA_REGISTRY_URL: https://kafka-schema-registry.nais-q.adeo.no/
 
 AIVEN_ES_HOST: elastic-klage-oppgave-es-nav-dev.aivencloud.com
 AIVEN_ES_PORT: 26482
+
+SAF_BASE_URL: https://saf.dev-fss-pub.nais.io/
+SAF_APP_CLIENT_ID: c7adbfbb-1b1e-41f6-9b7a-af9627c04998
+
+spring:
+  flyway:
+    clean-on-validation-error: true
```

Comment:
Denne trenger ikke å være der akkurat nå. Nå har vi samme strategi i dev og prod, og innstillingene ligger i application.yml.

Reply:
Dukket den opp som noe jeg hadde lagt til? Det var snålt, for det har jeg ikke gjort..

Reply:
Jeg tar den bort jeg da, uansett!